app/controllers/home_controller.rb

```diff
@@ -1,4 +1,13 @@
 class HomeController < ApplicationController
   def index
+    if current_user
+      if current_user.is_admin
+        redirect_to questions_path
+      else
+        redirect_to my_profile_path
+      end
+    else
+      redirect_to new_user_session_path
+    end
```

Comment:
Imbricated `if/else` statement is often confusing, and not really easy to read.
Thus, it encourage the add of another `if` branch when we'll need to add a new feature.
What do you think about this?

```
redirect_to(new_user_session_path) unless current_user
home_path = current_user.is_admin ? questions_path : my_profile_path
redirect_to(home_path)
```